Hema/test/power_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+require 'first_project/power'
+
+class PowerTest < TestBase
+  test 'powerofnumber' do
```

Comment:
since the name is just a string, i think that it is easier to read like this: :smile: 
```suggestion
  test 'power of number' do
```
And then when you run the test, it will convert it internally to `power_of_number` which is how we name methods in Ruby (this naming convention is called snake case)